app/javascript/components/hello_name.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function HelloName (props) {
+  const name = props.name;
+  return (
+    <div>
+        <h1>Hello {name}!</h1>
+    </div>
+  );
+}
+
+HelloName.propTypes = {
+  name: PropTypes.string.isRequired
+};
```

Comment:
This is something we can address using Typescript in your next ticket if you would like. Typescript provides a nice, robust way of ensuring props are present and of the correct type.

Reply:
Sounds good!